test/version_solver_test.dart

```diff
@@ -1040,7 +1040,62 @@ void dartSdkConstraint() {
     await expectResolves(environment: {'_PUB_TEST_SDK_VERSION': '2.0.0'});
   });
 
-  test('package deps disallow 2.0.0-dev by default', () async {
+  test('package deps allow 2.0.0-dev by default', () async {
+    await d.dir('foo', [
+      await d.pubspec({'name': 'foo'})
+    ]).create();
+    await d.dir('bar', [
+      await d.pubspec({'name': 'bar'})
+    ]).create();
+
+    await d.dir(appPath, [
+      await d.pubspec({
+        'name': 'myapp',
+        'dependencies': {
+          'foo': {'path': '../foo'},
+          'bar': {'path': '../bar'},
+        }
+      })
+    ]).create();
+
+    await expectResolves(
+      environment: {'_PUB_TEST_SDK_VERSION': '2.0.0-dev.99'},
+      // Log output should mention the PUB_ALLOW_RELEASE_SDK environment
+      // variable and mention the foo and bar packages specifically.
+      output: allOf(contains('PUB_ALLOW_PRERELEASE_SDK'),
+          anyOf(contains('foo, bar'), contains('bar, foo'))),
```

Comment:
Or you could just sort it? Might make it easier to parse for the user, too...

Reply:
Or just:

```dart
allOf(contains('PUB_ALLOW_PRERELEASE_SDK'), contains('foo'), contains('bar'))
```

Reply:
went ahead and sorted it since this is actually a long list in many cases so its a bit easier to parse